cnvlib/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.1.dev0"
+__version__ = "0.7.2.dev0"
```

Comment:
Is there an incantation for "later than the last stable version"? Some tools don't agree with the "+" suffix Biopython uses.